test/doctool/test-doctool-versions.js

```diff
@@ -2,8 +2,14 @@
 
 require('../common');
 const assert = require('assert');
+const { spawnSync } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
 const util = require('util');
-const { versions } = require('../../tools/doc/versions.js');
+
+const debuglog = util.debuglog('test');
+const versionsTool = path.join('..', '..', 'tools', 'doc', 'versions.js');
```

Comment:
`path.join('../../tools/doc/versions.js')` should yield the same result here and be platform safe, or maybe `path.resolve()` is what you want.